docs/antrea-l7-network-policy.md

```diff
@@ -146,7 +146,7 @@ spec:
             path: "/admin/*"
         - http:
             path: "/public/*"
-    - name: for-public   # Allow inbound HTTP GET requests to "/public" from Pods with label "app=client".
+    - name: for-public   # Allow inbound HTTP GET requests to "/public" from Pods.
```

Comment:
Based on the description above, I suppose the example is correct but the comment should be updated? Or we need to add "app=client" selector instead?

Reply:
it should probably be "from everyone"
it's not limited to traffic coming from Pods

Reply:
There is also a typo in the description above the example (not introduced by your PR):
s/while make other URLs publicly/while making other URLs publicly

Reply:
Reviewed and updated.